frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -205,6 +213,15 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
                             funnel_step: funnelStep,
                             ...(breakdown_value !== undefined && { funnel_step_breakdown: breakdown_value }),
                         }
+
+                        // getting property correlations from funnel
+                        if (params.funnel_custom_steps) {
+                            eventUsageLogic.actions.reportCorrelationInteraction(
+                                FunnelCorrelationResultsType.Properties,
+                                'person modal',
```

Comment:
Non-blocking future improvement: We should send a source in `insight person modal viewed` and set this to the relevant analysis view.